libraries/report.rb

```diff
@@ -61,8 +55,6 @@ class ComplianceReport < Chef::Resource
           else
             Chef::Log.warn "#{collector} is not a supported inspec report collector"
           end
-
-          raise "#{total_failed} audits have failed.  Aborting chef-client run." if total_failed > 0 && node['audit']['fail_if_any_audits_failed']
```

Comment:
Hmm. Since [this is the only use](https://github.com/chef-cookbooks/audit/search?utf8=%E2%9C%93&q=fail_if_any_audits_failed), at least the attribute should go away. That said, can't we aggregate in the cookbook code?